tests/commands/timesource.py

```diff
@@ -0,0 +1,100 @@
+#
+# Martin Kolman <mkolman@redhat.com>
```

Comment:
Same comment about emails in copyright messages

Reply:
Removed.